javascript-bank/account.js

```diff
@@ -0,0 +1,50 @@
+/* exported Account */
+function Account(number, holder) {
+  this.number = number;
+  this.holder = holder;
+  this.transactions = [];
+}
+
+Account.prototype.deposit = function (amount) {
+  if (isNaN(amount)) {
+    return false;
+  } else if (!Number.isInteger(amount)) {
+    return false;
```

Comment:
Technically, `Number.isInteger` will also catch cases where `amount` is not a number.